server/controllers/wazuh-api.ts

```diff
@@ -1060,7 +1060,7 @@ export class WazuhApiCtrl {
       
       const disabledRoles = ( await getConfiguration() )['disabled_roles'] || [];
       const logoSidebar = ( await getConfiguration() )['customization.logo.sidebar'] || 'icon_blue.png';
-      const wazuhSecurity = SecurityObj(context.wazuh.plugins);
+      const wazuhSecurity = await SecurityObj(context.wazuh.plugins, context);
```

Comment:
**nitpick**: use the `context.wazuh.security` instead of create another security object.